closed/NVIDIA/docker/Dockerfile

```diff
@@ -49,12 +49,14 @@ ENV LANG=C.UTF-8
 
 ### Install necessary python 2 modules
 RUN python2 -m pip install --upgrade pip \
+ && python2 -m pip install setuptools \
  && python2 -m pip install --upgrade setuptools wheel virtualenv \
```

Comment:
@vilmara Shouldn't this line already install `setuptools`?

Reply:
The line `python2 -m pip install --upgrade setuptools wheel virtualenv` doesn't install setuptools, it should install it but is not working properly; so I added the additional commands to install setup tools and fix the error "ImportError: No module named setuptools"

Reply:
Yeah, I guess adding this doesn't hurt, anyway. Thanks for fixing this